readme_developers.md

```diff
@@ -0,0 +1,53 @@
+   git lfs pull
+   ```
+ - when downloading Unity (current version for this project is 2019.4.20.f1) choose modules according to your needs (e.g. Android build support for creating .apk files)
+ - get third party assets from Unity AssetStore (extract and copy into Assets/Submodules):
+   - [Lunar Mobile Console](https://assetstore.unity.com/packages/tools/gui/lunar-mobile-console-free-82881) (optional)
+   - [Loading screen animation](https://assetstore.unity.com/packages/tools/loading-screen-animation-98505) 
+   - [Modern UI Pack](https://assetstore.unity.com/packages/tools/gui/modern-ui-pack-150824)
+   - [UI Graph](https://assetstore.unity.com/packages/tools/gui/ui-graph-51304)
+   - [Joystick Pack](https://assetstore.unity.com/packages/tools/input-management/joystick-pack-107631)
+   - [Native Camera for Android & iOS](https://assetstore.unity.com/packages/tools/integration/native-camera-for-android-ios-117802) (only if you want to build Android version)
+   - [Trilib 2](https://assetstore.unity.com/packages/tools/modeling/trilib-2-model-loading-package-157548) - manually delete folder Trilib/Plugins/NewtonSoft.Json/ otherwise, there will be most likely conflicts
+   - [Simple Collada](https://assetstore.unity.com/packages/tools/input-management/simple-collada-19579)
+     - requires to apply patch (<b>/scripts/ColladaImporter.patch</b> and <b>/scripts/StringExtensions.patch</b>) on script ColladaImporter.cs and OrbCreationExtensions/StringExtensions.cs
+     - SimpleCollada asset must be moved to Assets/Submodule folder in order to patch script works
+     - on Windows, you can download [UnxUtils](http://unxutils.sourceforge.net/) and use batch file in /scripts/patch_simple_collada_asset.bat (if UnxUtils extracted to "C:\Program Files\") or use this command:
+     ```bash       
+     path_to_UnxUtils\UnxUtils\usr\local\wbin\patch.exe ..\arcor2_AREditor\Assets\Submodules\SimpleCollada\ColladaImporter.cs -i ColladaImporter.patch
+     path_to_UnxUtils\UnxUtils\usr\local\wbin\patch.exe ..\arcor2_AREditor\Assets\Submodules\SimpleCollada\OrbCreationExtensions\StringExtensions.cs -i StringExtensions.patch
+     ```
+     - on Linux, you can use bash script in /scripts/patch_simple_collada_asset.sh
+
+ - download and set custom gradle - follow https://developers.google.com/ar/develop/unity/android-11-build#unity_20193_and_20194 (We recommend "binary-only" version of gradle, since we encountered bugs with "complete" version)
+ - delete folder repository\arcor2_AREditor\Assets\Submodules\TriLib\TriLibSamples
+ - in Unity, go to Project Settings -> XR -> ARCore Extensions and change Android Authentication Strategy from "Keyless" to "Api Key"
+
+### Building
+ - when building for Android: go to Project Settings -> Player -> Other Settings and add "AR_ON" to Scripting Define Symbols
+ - when building for Unity editor (no matter which platform): Remove "AR_ON" from Scripting Define Symbols in Project Settings -> Player -> Other Settings
```

Comment:
When the platform is set to Standalone, AR_ON does not have to be removed and it is ignored. When the platform is set to Android, if you build the .apk or want to use AR Foundation Remote, the AR_ON has to be set, whilst when running play mode without the tablet, AR_ON has to be removed.

Reply:
readme updated